var/spack/repos/builtin/packages/atmi/package.py

```diff
@@ -112,7 +114,8 @@ class Atmi(CMakePackage):
     root_cmakelists_dir = "src"
 
     patch("0001-Remove-relative-link-paths-to-external-libraries.patch", when="@3.5.0")
-    patch("0002-Remove-usr-bin-rsync-reference.patch", when="@4.0.0:")
+    patch("0002-Remove-usr-bin-rsync-reference.patch", when="@4.0.0:5.1.3")
+    patch("0002-Remove-usr-bin-rsync-reference-5.2.0.patch", when="@5.2.0:")
```

Comment:
Can we fix this upstream instead of maintaining patches in Spack for different version ranges? In any case, can you add comments above these directives to explain what the patch does, why it is needed and where it was taken from?

Reply:
Added header in the patch file and provided the comment in the recipes.

Reply:
> Added header in the patch file and provided the comment in the recipes.

Hmm, I don't see any comment above the `patch` directive